src/handlers/proposalVersionsHandlers.ts

```diff
@@ -212,6 +214,30 @@ const postProposalVersion = (
 		});
 };
 
+const getProposalVersion = (
+	req: Request,
+	res: Response,
+	next: NextFunction,
+): void => {
+	const { proposalVersionId } = req.params;
+	if (!isId(proposalVersionId)) {
+		next(new InputValidationError('Invalid request body.', isId.errors ?? []));
```

Comment:
This is a copy pasted error (as in, the error exists in another endpoint so I totally feel for you) -- proposalVersionId is a query parameter, not part of the request body, so this error should be updated to say `invalid query parameter`

Reply:
Nice catch!